packages/env/lib/config/test/config.js

```diff
@@ -101,13 +102,17 @@ describe( 'readConfig', () => {
 			process.env.WP_ENV_HOME = 'here/is/a/path';
 			const configWith = await readConfig( '.wp-env.json' );
 			expect(
-				configWith.workDirectoryPath.includes( 'here/is/a/path' )
+				configWith.workDirectoryPath.includes(
+					`here${ sep }is${ sep }a${ sep }path`
```

Comment:
I think it'd be easier to use `path.join`:
```suggestion
					join( 'here', 'is', 'a', 'path' )
```